mimic/core.py

```diff
@@ -82,11 +99,19 @@ def service_with_region(self, region_name, service_id, base_uri):
         """
         if service_id in self._uuid_to_api:
             api = self._uuid_to_api[service_id]
-            return api.resource_for_region(
-                region_name,
-                self.uri_for_service(region_name, service_id, base_uri),
-                self.sessions,
-            )
+            if IExternalAPIMock.providedBy(api):
+                # External APIs only provide a catalog entry, nothing more
+                raise RuntimeError(
+                    api.__class__.__module__ + '/' +
+                    api.__class__.__name__ +
+                    " is an external service to Mimic."
+                )
+            else:  # IAPIMock.providedBy(api)
```

Comment:
I think the comment here should be a bit more expository; what I believe it's trying to express is "we already validated that it implements one or the other of these in `__init__`"?